include/sundials/sundials_types.h

```diff
@@ -94,8 +94,8 @@ typedef float sunrealtype;
 #define SUN_BIG_REAL      FLT_MAX
 #define SUN_SMALL_REAL    FLT_MIN
 #define SUN_UNIT_ROUNDOFF FLT_EPSILON
-#define SUN_REAL_FORMAT_E "%." SUN_STRING(FLT_DIG) "e"
-#define SUN_REAL_FORMAT_G "%." SUN_STRING(FLT_DIG) "g"
+#define SUN_FORMAT_E      "%." SUN_STRING(FLT_DIG) "e"
+#define SUN_FORMAT_G      "%." SUN_STRING(FLT_DIG) "g"
```

Comment:
The single and double formats to not have a space between the `%` and `.` while the long double format does. I think having the space is better as it will align positive and negative number better.